src/main/fc/settings.yaml

```diff
@@ -49,6 +49,9 @@ tables:
   - name: gps_dyn_model
     values: ["PEDESTRIAN", "AIR_1G", "AIR_4G"]
     enum: gpsDynModel_e
+  - name: gps_ublox_use_galileo
+    values: ["OFF", "ON"]
+    enum: gpsUbloxUseGalileo_e
```

Comment:
Can you remove this table? It seems it does nothing now.

Reply:
I think the error is that gps_ublox.c no longer references the related configure variable, which was not intended when I changed the criteria.

Reply:
If I'm understanding it correctly, a boolean without any table seems like the best option. Do you plan to add more states to the variable in the future?